src/package_io/input_parser.star

```diff
@@ -34,7 +34,7 @@ HIGH_DENEB_VALUE_FORK_VERKLE = 20000
 
 # MEV Params
 FLASHBOTS_MEV_BOOST_PORT = 18550
-MEV_BOOST_SERVICE_NAME_PREFIX = "mev-boost-"
+MEV_BOOST_SERVICE_NAME_PREFIX = "mev-boost"
```

Comment:
do we not need the extra `-`?

Reply:
added EL_CL name to the mev boost, see below,